### DISCUSSION TITLE
Perftest, where?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#867] [General]
With regard to https://github.com/GrandOrgue/grandorgue/discussions/862 and other threads, where can I find the perftest application? My OS i Ubuntu 20.04 LTS.

**Comment by oleg68** [comment]
There is no such application now. I created an issue #868 for shipping this application with the GO distribution

**Reply by jason-barba** [reply]
Your answer confuses me. In #862 is said, I qoute: "the perftest application that comes with GO".

**Reply by oleg68** [reply]
This page is under construction now. There is an obsolete information there.
